app/controllers/avo/associations_controller.rb

```diff
@@ -78,7 +80,9 @@ def create_association
       association_name = BaseResource.valid_association_name(@record, association_from_params)
 
       perform_action_and_record_errors do
-        if has_many_reflection?
+        if through_reflection? && additional_params?
+          new_join_record.save
+        elsif has_many_reflection?
```

Comment:
```suggestion
        elsif has_many_reflection? || through_reflection?
```

Let's ensure that `@record.send(association_name) << @attachment_record` is executed when `through_reflection?` but there are no additional params